docs/user-guide/global-configurations/deployment-charts.md

```diff
@@ -1,70 +1,94 @@
 # Deployment Charts
 
-Devtron includes predefined helm charts that cover the majority of use cases.
-For any use case not addressed by the default helm charts, you can upload your own helm chart and use it as a custom chart in Devtron.
+## Introduction
 
-* Who can upload a custom chart - Super admins
-* Who can use the custom chart - All users
-
-> A super admin can upload multiple versions of a custom helm chart.
+Devtron includes predefined Helm charts that cover the majority of use cases.
+For any use case not addressed by the default Helm charts, you can upload your own Helm chart and use it as a deployment chart in Devtron.
 
 ![Figure 1: Deployment Charts](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-charts/gc-deployment-charts.jpg)
 
-## Prerequisites
+### Tutorial
+
+This video contains a quick walkthrough of the steps mentioned in the [Prerequisites](#prerequisites) section of this page and the subsequent uploading of the deployment chart on Devtron.
+
+{% embed url="https://www.youtube.com/watch?v=jzMZa7bSiyA" caption="How to Upload your Deployment Chart in Devtron" %}
+
+---
 
-1. A valid helm chart, which contains `Chart.yaml` file with name and version fields.
-2. Image descriptor template file `.image_descriptor_template.json`.
-3. Custom chart packaged in the `*.tgz` format.
+## Prerequisites
 
-### 1. How to create a helm chart
+### 1. Create a Helm Chart
 
-You can use the following command to create the Helm chart:
+You can use the following command to create a Helm chart:
 
 ```bash
 helm create my-custom-chart
 ```
 
->Note: `Chart.yaml` is the metadata file that gets created when you create a [helm chart](https://helm.sh/docs/helm/helm_create/).
+> **Note**: `Chart.yaml` is the metadata file that gets created when you create a [helm chart](https://helm.sh/docs/helm/helm_create/). The following table consists the fields that are relevant to you in `Chart.yaml`.
 
 | Field | Description |
 | --- | --- |
-| `Name` | Name of the helm chart (Required). |
+| `Name` | Name of the Helm chart (Required). |
 | `Version` | This is the chart version. Update this value for each new version of the chart (Required). |
-| `Description` | Description of the chart (Optional). |
+| `Description` | Give a description to your chart (Optional). |
 
-Please see the following example:
+{% hint style="info" %}
+### Example of Chart.yaml
+[Click here](https://devtron-public-asset.s3.us-east-2.amazonaws.com/custom-charts/chart-yaml-file.png) to view a sample `Chart.yaml` file.
+{% endhint %}
 
-![Chart.yaml file](https://devtron-public-asset.s3.us-east-2.amazonaws.com/custom-charts/chart-yaml-file.png)
+### 2. Create an Image Descriptor Template File
```

Comment:
why an image descriptor is required?

Reply:
Was already explained few lines below, but enhanced it now